sorc/machine-setup.sh

```diff
@@ -56,16 +56,16 @@ elif [[ $hname = 'hpc.msstate.edu' ]] ; then
 elif [[ -d /scratch1 ]] ; then
     # We are on NOAA Hera
     if ( ! eval module help > /dev/null 2>&1 ) ; then
-	echo load the module command 1>&2
+        echo load the module command 1>&2
         source /apps/lmod/lmod/init/$__ms_shell
     fi
-    target=hera
+    target=orion
```

Comment:
This section should still be for Hera, no?

Reply:
(there are more changes to orion on subsequent lines that will need to be changed back)

Reply:
Thanks @WalterKolczynski-NOAA I'll take a look when I get a chance.  I'm guessing this also means this PR should be tested on all the machines before merging as well to make sure other platform dependent bugs don't creep in unnoticed as it's tons of changes.

Reply:
@WalterKolczynski-NOAA I reverted back to what was in feature/coupled-crow. 

Any other red-flags for this branch or are we just at the stage where this branch needs to be run on all platforms?

Reply:
I think just testing. Looked pretty good overall.